frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -667,6 +672,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [OCS_FLAG],
     },
   },
+  // In case Noobaa is deployed without OCS Operator
```

Comment:
Why do we need to do this ?
Can't the respective tabs extensions be guarded ?

Reply:
The situation is not so simple. 
Just MCG deployment will not have OCS. So in that case we need to show only the MCG dashboard. Without the block and file. Tab extension guard will not be enough.